Framework/Algorithms/src/DetectorEfficiencyCor.cpp

```diff
@@ -245,9 +251,9 @@ void DetectorEfficiencyCor::correctForEfficiency(
     // now get the sin of the angle, it's the magnitude of the cross product of
```

Comment:
Using `detectorInfo.twoTheta(detIndex)` causes the unit tests to fail because the source isn't set in the instrument. I guess the current code gets around that as it doesn't use the source. I'm not sure whether that's a good thing or not! Any thoughts?